packages/client/src/app/app-routing.module.ts

```diff
@@ -46,10 +44,12 @@ const routes: Routes = [
     {path: 'portfolio/:id/:username', resolve: {portData: PortfolioResolver}, runGuardsAndResolvers: 'always', component: PortfolioComponent, children: [
       {path: 'blog', component: PortBlogComponent, resolve: {feedData: PortBlogsResolver}, runGuardsAndResolvers: 'always'},
       {path: 'blog/:blogId', resolve: {blogData: BlogPageResolver}, runGuardsAndResolvers: 'always', component: PortBlogPageComponent},
-      {path: 'works', component: PortWorksComponent, resolve: {feedData: PortWorksResolver}, runGuardsAndResolvers: 'always'},
+      {path: 'works', component: WorksComponent, resolve: {feedData: PortWorksResolver}, runGuardsAndResolvers: 'always'},
```

Comment:
Just noting that the naming convention is inconsistent. Some components and feedDatas both start with "Port", some don't.

Reply:
yeah, with this update, a lot of names and components will be normalized so as not to cause as many conflicts